ui/src/alerting/reducers/checks.ts

```diff
@@ -66,3 +66,7 @@ export default (
         return
     }
   })
+
+export const getCheckIDs = (state: ChecksState): {[x: string]: boolean} => {
```

Comment:
This would be more similar to most of the other selectors in the app if it lived in a `selectors` directory and accepted an `AppState` rather than `ChecksState` as an argument.

Reply:
Ah, yes they should take all of AppState